lib/Property/ICalendar/Recur.php

```diff
@@ -54,7 +54,7 @@ function setValue($value) {
                     if (strcmp($k, 'until') === 0) {
                         $v = strtr($v, [':' => '', '-' => '']);
                     }
-                } else {
+                } elseif (!is_int($v)) {
```

Comment:
To add to that previous conversation, it might simply be a bit better to use `is_array` and not `is_int`. I think it makes this a bit clearer. Other than that, patch looks good!